kong/plugins/ai-prompt-guard/schema.lua

```diff
@@ -0,0 +1,50 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+return {
+  name = "ai-prompt-guard",
+  fields = {
+    { protocols = typedefs.protocols_http },
+    { config = {
+      type = "record",
+      fields = {
+          { allow_patterns = {
+              description = "Array of valid patterns, or valid questions from the 'user' role in chat.",
+              type = "array",
+              default = {},
+              elements = { type = "string" } } },
```

Comment:
Please add a limit to the string size here. This is to avoid huge strings in configuration. Kong hasn't historically done it but we would like to follow it in new development.

Reply:
Also put a limit on the number of array items. These limits should be conservative and can be expanded in the future.